<?xml version='1.0' encoding='UTF-8'?>
<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en">
  <id>https://selfh.st/apps</id>
  <title>Syncthing Releases</title>
  <updated>2025-04-02T05:31:41.842171-04:00</updated>
  <author>
    <name>selfh.st</name>
    <email>contact@selfh.st</email>
  </author>
  <link href="https://selfh.st/apps" rel="alternate"/>
  <generator uri="https://lkiesow.github.io/python-feedgen" version="0.9.0">python-feedgen</generator>
  <subtitle>Self-hosted software releases generated by selfh.st</subtitle>
  <entry>
    <id>https://github.com/syncthing/syncthing/releases/tag/v1.29.0</id>
    <title>New release for Syncthing: v1.29.0</title>
    <updated>2025-01-06T03:26:32-05:00</updated>
    <author>
      <name>syncthing/syncthing</name>
    </author>
    <content>&lt;p&gt;v1.29.0&lt;/p&gt;
&lt;p&gt;Bugfixes:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9855: GUI address unix socket broken&lt;/h1&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;h1&gt;9872: When socket files are in a folder that needs to be synced on linux, some files were not synced until the socket files were removed&lt;/h1&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;h1&gt;9875: Clarify log message for ignored folders&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;
&lt;p&gt;Other issues:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9870: Fails with Go 1.24: runtime go1.24rc1 not found in compat.yaml&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;</content>
    <link href="https://github.com/syncthing/syncthing/releases/tag/v1.29.0" rel="alternate"/>
  </entry>
  <entry>
    <id>https://github.com/syncthing/syncthing/releases/tag/v1.29.1</id>
    <title>New release for Syncthing: v1.29.1</title>
    <updated>2025-01-09T16:11:28-05:00</updated>
    <author>
      <name>syncthing/syncthing</name>
    </author>
    <content>&lt;p&gt;v1.29.1&lt;/p&gt;
&lt;p&gt;Bugfixes:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9904: The browse API response has changed as of Syncthing v1.29.0&lt;/h1&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;h1&gt;9906: Deadlocks randomly, occasionally&lt;/h1&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;h1&gt;9907: The response of "pending" APIs has changed as of Syncthing v1.29.0&lt;/h1&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;h1&gt;9909: Login into UI (Browser) not working anymore&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;</content>
    <link href="https://github.com/syncthing/syncthing/releases/tag/v1.29.1" rel="alternate"/>
  </entry>
  <entry>
    <id>https://github.com/syncthing/syncthing/releases/tag/v1.29.2</id>
    <title>New release for Syncthing: v1.29.2</title>
    <updated>2025-01-12T10:45:05-05:00</updated>
    <author>
      <name>syncthing/syncthing</name>
    </author>
    <content>&lt;p&gt;v1.29.2&lt;/p&gt;
&lt;p&gt;Bugfixes:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9913: panic: string field contains invalid UTF-8 [recovered]&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;</content>
    <link href="https://github.com/syncthing/syncthing/releases/tag/v1.29.2" rel="alternate"/>
  </entry>
  <entry>
    <id>https://github.com/syncthing/syncthing/releases/tag/v1.29.3</id>
    <title>New release for Syncthing: v1.29.3</title>
    <updated>2025-03-12T07:56:30-04:00</updated>
    <author>
      <name>syncthing/syncthing</name>
    </author>
    <content>&lt;p&gt;v1.29.3&lt;/p&gt;
&lt;p&gt;Bugfixes:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9929: Unintelligible error message "setting metadata: lookup failed"&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;</content>
    <link href="https://github.com/syncthing/syncthing/releases/tag/v1.29.3" rel="alternate"/>
  </entry>
  <entry>
    <id>https://github.com/syncthing/syncthing/releases/tag/v1.29.4</id>
    <title>New release for Syncthing: v1.29.4</title>
    <updated>2025-04-01T04:45:07-04:00</updated>
    <author>
      <name>syncthing/syncthing</name>
    </author>
    <content>&lt;p&gt;v1.29.4&lt;/p&gt;
&lt;p&gt;Bugfixes:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;
&lt;h1&gt;9990: Tildes break directory autocomplete&lt;/h1&gt;
&lt;/li&gt;
&lt;/ul&gt;</content>
    <link href="https://github.com/syncthing/syncthing/releases/tag/v1.29.4" rel="alternate"/>
  </entry>
</feed>
